.storybook/recipes/TableCard/TableCard.tsx

```diff
@@ -75,20 +75,14 @@ export const TableCard = ({
         >
           {title}
         </Heading>
-        <Table
-          caption="Standards coverage"
-          className={styles['table-card__table']}
-          hideCaption={true}
-        >
+        <Table className={styles['table-card__table']} title={title}>
           <TableHeader>
-            <TableRow>
+            <TableRow variant="header">
```

Comment:
`header` variant for `TableRow` has darker bottom border color to distinguish itself as header